### DISCUSSION TITLE
لیست کشته شده ها، دستگیر شده ها و مزدوران

### DISCUSSION THREAD

**Discussion by ashahi000** [WeAreMahsaAmini/Core#436] [Ideas]
جمع آوری لیستی از اسامی و اطلاعات افراد کشته شده، دستگیر شده و مزدوران از خبرهای متفرقه. 
میتونه لیست عمومی باشه و همه ببینن و اشخاص کامنت بذارن.
میتونه بیست شامل اطلاعات دستیگیری، تاریخ  و شرح واقعه باشد که فردای پیروزی برای پیگیری، مستندات از خانواده ها هم جمع آوری شود و قابل پیگیری شود.

**Comment by masoodsh77** [comment]
چطور میشه بهش اضافه کرد اسم افراد رو ؟

**Reply by masoodsh77** [reply]
من فرانت کار هستم (ری اکت ) 
میتونم توی این پروژه ها کمکتون کنم

**Reply by adeleshahi** [reply]
من فعلا یه گوگل شیت درست کردم که قابلیت اشتراک گذاری عمومی داره. منتها نمیدونم چقدر میشه به امنیت اون مطمین بود. الان لیست من ناکامل هست ولی تا همین الان در مجموع 200 نفر رو جمع آوری کرده ام.
